descriptors/ModuleDescriptor-template.json

```diff
@@ -365,6 +365,14 @@
             "inventory-storage.instances.item.get"
           ]
         },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/inventory/tenant-items",
+          "permissionsRequired": ["inventory.items.collection.get"],
```

Comment:
we need to create new permission for each endpoint instead of reusing existing. Also as I know Eureka platform does not support assigning the same permission for multiple endpoints

Reply:
resolved